Docs/Configuration.tex

```diff
@@ -1495,7 +1494,7 @@ \subsection{Quirks Properties}\label{booterpropsquirks}
   into the unavailable range. When \texttt{AppleDebug} is enabled, usually the
   debug log may contain messages like \texttt{AAPL: [EB|`LD:LKC] \} Err(0x9)}.
   To find the optimal value, manually append \texttt{slide=X} to \texttt{boot-args}
-  and log the largest one that won't cause boot failure.
+  and log the largest one that will not result in boot failures.
```

Comment:
Still, failure is uncountable.

Reply:
As earlier response

Reply:
Right. Then it can be left either way I guess, and I am fine with it. :)